cmake/licenses.cmake

```diff
@@ -29,6 +29,17 @@
 
 include(CMakeParseArguments)
 
+define_property(
+    DIRECTORY PROPERTY IVW_LICENCE_LIST
```

Comment:
British English or American English spelling? Compare also with filename

Reply:
change to license